src/Components/MainHeader/MainHeader.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+import './MainHeader.css';
+
+class MainHeader extends Component {
+  render() {
```

Comment:
use deconstruction for two and more props
`const {var1,var2} = this.props`

in this cast better use stateless component and deconsturction use in brackets '({var1, var2})'

and added propsType